### DISCUSSION TITLE
Is WDD compatible with Windows 2H24 update?

### DISCUSSION THREAD

**Discussion by tjsuominen** [t1m0thyj/WinDynamicDesktop#614] [Q&A]
Microsoft has put some W11 machines on hold receiving the Windows 2H24 update due to 'incompatibilities with some wallpaper apps'.

Is WDD compatible with 2H24 update? 

Reference: https://learn.microsoft.com/en-us/windows/release-health/status-windows-11-24h2#3370msgdesc

**Comment by t1m0thyj** [comment]
This is interesting news. I have not yet upgraded to Windows 11 24H2 or received any communication from Microsoft about this so I can only speculate.

> As new versions and updates are released for these applications by their respective developers, it's expected that these issues will be resolved.

I'm not aware of any changes to the system APIs used by WinDD to update wallpapers that would result in an incompatibility.

> Multiple applications are affected, and symptoms can include wallpapers not appearing correctly

My primary machine doesn't have the 24H2 update yet but I did some basic testing in a VM running Windows 11 Insiders build 26000.1 and WinDD ran fine.

So I would assume WinDD is not impacted, but I have not extensively tested 24H2. If you decide to upgrade, please share your results 😁

**Reply by t1m0thyj** [reply]
I noticed there has been an issue reported by some users on 24H2: #601

It seems that the symptoms appear only after resuming from sleep. This is likely a Windows bug but I'll keep monitoring it.
